src/main.cc

```diff
@@ -0,0 +1,84 @@
+
+#include <cstdlib> // For EXIT_SUCCESS, EXIT_FAILURE
+#include <cstring> // For strcmp
```

Comment:
bro strcmp has caused some of the biggest hacks in software history, dont use C functions